REST API and UI Collaboration/README.md

```diff
@@ -1,7 +1,73 @@
-# REST API and UI Web App
+# GodelTech.Microservices.WebAndApiCollaboration
+
+## Overview
+
+**GodelTech.Microservices.WebAndApiCollaboration** is example of system that includes following interacting parts:
 
-This example contains the following interacting parts:
 - REST API Website
-- UI Website based on Razor Pages performned API call
-- Indentity Server responsible for authentication
+- UI Website based on Razor Pages performed API call
+- Identity Server responsible for authentication and authorization
+
+All these parts is based on **GodelTech.Microservice.Framework**.
+
+Expected use case for this application is the web application that interacts with protected resources on behalf of authenticated user.
+
+## GodelTech.Microservices.WebAndApiCollaboration.Web
+
+This is the client which performs Api calls on behalf of users.
+
+There are two main use cases in this application to show authentication and authorization processes.
+
+Use case 1:
```

Comment:
I would highlight this line using header of third level (e.g. ###)

Reply:
Use case 1 was highlighted